types/v56/openapi.go

```diff
@@ -50,3 +50,69 @@ type Role struct {
 	BundleKey   string `json:"bundleKey"`
 	ReadOnly    bool   `json:"readOnly"`
 }
+
+// NsxtTier0Router defines NSX-T Tier 0 router
+type NsxtTier0Router struct {
+	ID          string `json:"id,omitempty"`
+	Description string `json:"description"`
+	DisplayName string `json:"displayName"`
+}
+
+// ExternalNetworkV2 defines a struct for OpenAPI endpoint
+type ExternalNetworkV2 struct {
+	ID              string                    `json:"id,omitempty"`
+	Name            string                    `json:"name"`
+	Description     string                    `json:"description"`
+	Subnets         ExternalNetworkV2Subnets  `json:"subnets"`
+	NetworkBackings ExternalNetworkV2Backings `json:"networkBackings"`
```

Comment:
I think would be good to have comment with possible values

Reply:
Added more docs whatever I could find